### PR TITLE
Touch feedback added.

### PR DESCRIPTION
**Pull request by m-murad**
Solves issue #76 

@chhavip @anubhakushwaha @yatna
please review the PR.

### CODE REVIEW

**Comment by anubhakushwaha** [comment]
@free4murad Please check,the app is crashing in tripIndicatorFragmentactivity

File: malaria-app-android/src/main/res/layout/rapid_fire.xml [hunk 1]
```diff
@@ -135,6 +135,7 @@
             android:paddingRight="4dp"
             android:paddingLeft="4dp"
             android:textSize="14.0sp"
-            android:onClick="exitGame"/>
+            android:onClick="exitGame"
+            android:id="@+id/exit" />
```
Comment (anubhakushwaha) [inline malaria-app-android/src/main/res/layout/rapid_fire.xml]: id is usually the first line,change it

**Comment by free4murad** [comment]
@anubhakushwaha 
I have made the changes, please review.

**Comment by DipanshKhandelwal** [comment]
@murad I know it's old but do we need it now? Like why to add more code if we get it automatically in material design ?

**Comment by geekanamika** [comment]
It will be solved while shifting application to material design via @haroonshah2764 's work.

**Pull request closed without merging**